Monika After Story/game/script-topics.rpy

```diff
@@ -13016,6 +13016,43 @@ label monika_unknown:
         m 1hubfa "I love you, [player]. I'll always be rooting for you~"
     return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel='monika_althistory',
+            prompt="Alternate history",
+            category=['literature','media'],
+            random=True
+        )
+    )
+
+label monika_althistory:
+    m 1esa "Say, [player], are you familiar with the concept of alternate history?"
+    m 1eua "It's a genre of speculative fiction that focuses on 'what if' scenarios where a certain event at a critical point of history happened differently."
+    m 3eua "While the genre's popularity has surged only recently, the first known example of alternate history speculation dates back to the times of Ancient Rome..."
+    m 3eud "A Roman historian named Livy contemplated a scenario in his history book, '{i}Ab Urbe Condita Libri{/i}', where Alexander the Great had surived his fatal fever."
+    m 2eud "A hypothetical scenario would have him pursue his plans of European expansion and inevitable confrontation of the Argead Empire with the Romans..."
+    m 2euc "Livy ultimately concluded that this scenario would likely end up becoming a Roman victory."
+    m 3eua "One of the better known contemporary examples is Robert Harris' detective novel '{i}Fatherland{/i}' which depicts a world where the Third Reich won the Second World War."
+    m 1eud "The book is set in alternate 1964 and follows the story of a Kripo detective who is investigating the murder of a high-ranking Nazi Party official..."
+    m 3euc "As he uncovers more details, he realizes that he is caught up in a political scandal involving senior party officials who are apparently being murdered under staged circumstances."
+    m 1eud "He ultimately discovers the truth behind the murders and the culprit is revealed to be Reinhard Heydrich, now leader of the SS. And the kicker? He's supposed to be long dead."
```

Comment:
I really don't know if Monika would say "kicker". It doesn't sound in character for me.

Reply:
No longer relevant due to rewrite.